The target-smp-command test is expected to fail. See upstream issue
https://github.com/espressif/openocd-esp32/issues/397
--- a/testing/tcl_commands/Makefile.am	2026-08-05 15:39:14.873577795 +0200
+++ b/testing/tcl_commands/Makefile.am	2026-08-05 15:39:34.500717294 +0200
@@ -5,8 +5,7 @@ TESTS =
 if DUMMY
 TESTS += \
 	test-target-create-command.cfg \
-	test-target-configure-cget-command.cfg \
-	test-target-smp-command.cfg
+	test-target-configure-cget-command.cfg
 endif
 
 EXTRA_DIST = utils.tcl $(TESTS)
